plugins/scaffolder-backend/sample-templates/remote-templates.yaml

```diff
@@ -7,3 +7,9 @@ spec:
   type: url
   targets:
     - https://github.com/spotify/cookiecutter-golang/blob/master/template.yaml
+    - https://github.com/backstage/software-templates/blob/master/scaffolder-templates/create-react-app/template.yaml
```

Comment:
I'm a little concerned of always pointing to the remote software-templates and them not being tied to a particular commit. The benefit of having them locally as it that they were versioned with the current version of the scaffolder, so it made it easier to make breaking changes as nothing depended on them.

Reply:
Pointing to a specific commit makes sense to me!

Reply:
We've also added a note in the software-templates README suggesting users fork these sample templates instead of referencing them directly

Reply:
Hmm, really? Pointing to a specific commit seems dangerous in that we can easily forget to update these references and get strange bug reports if we fix template problems. I think the README update is sufficient to express that the examples are provided as-is, expect breaking changes if you don't fork, etc. Maybe we should update the note in this file to echo the same:

```yaml
# Backstage example templates; for demonstration purposes only, these are not versioned and may receive changes incompatible with previous scaffolder versions
```

Reply:
We could also make it extremely clear and encourage forking by putting "sample" or "example" in the template names, like "Create React App Example"